src/averageAges.js

```diff
@@ -15,11 +15,14 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const filteredMan = (century !== undefined)
+    ? people.filter(person =>
+      Math.ceil(person.died / 100) === century
+      && person.sex === 'm'
+    )
+    : people.filter(person => person.sex === 'm');
```

Comment:
Still can be simplified, use ternary operator inside filter method, instead of writing filter method for both cases

Reply:
Fix everywhere please